actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -291,7 +298,7 @@ private void updateServerNodes()
 
     private Task<NodeAddress> locateActiveActor(final RemoteReference<?> actorReference)
     {
-        final NodeAddress address = localAddressCache.get(actorReference);
+        final NodeAddress address = localAddressCache.asMap().get(actorReference);
```

Comment:
Right... I thought computeIfPresent would throw null pointers, but getIfPresent is correct